src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -93,18 +94,70 @@ def export_group_as_template(
 
 def deploy_arm_template(
         resource_group_name, template_file=None, template_uri=None, deployment_name=None,
-        parameters=None, mode='incremental', no_wait=False):
+        parameters=None, mode='incremental', no_wait=False, prompt_for_parameters=False):
     return _deploy_arm_template_core(resource_group_name, template_file, template_uri,
-                                     deployment_name, parameters, mode, no_wait=no_wait)
+                                     deployment_name, parameters, mode, no_wait=no_wait,
+                                     prompt_for_parameters=prompt_for_parameters)
 
 def validate_arm_template(resource_group_name, template_file=None, template_uri=None,
-                          parameters=None, mode='incremental'):
+                          parameters=None, mode='incremental', prompt_for_parameters=False):
```

Comment:
@tjprescott @derekbekoe requested `--prompt` here: https://github.com/Azure/azure-cli/pull/2364#discussion_r104485911

I don't care which we do, but you folks should agree between yourselves and then let me know..